src/templates/docs/white-space/index.html

```diff
@@ -47,7 +47,8 @@ <h4 class="f4 mv0 fw6 dib mr4"><%= name %></h4>
       <div class="ph3 ph5-ns pt4 pb5">
         <h2 class="f4 bold">Examples</h2>
       <p class="f5 f4-ns measure lh-copy">This text is formatted the same way in the source and printed three times, with the only difference being which white-space class is attached.</p>
-        <h3 class="f5 book pt4 code">.ws-normal</h3>
+        <h3 class="f5 fw4 pt4 ttu tracked mb1">Normal</h3>
```

Comment:
Was missing titles for example so added them in, may not be correct styles though